data/transport missions.txt

```diff
@@ -1277,6 +1277,7 @@ mission "Drug Running 1"
 	on complete
 		payment
 		payment 80000
+		"drug smuggler" ++
```

Comment:
What is this drug smuggler condition for, seeing as how you haven't used it anywhere? And should this be used, why not have `"drug smuggler" ++` on drug running pirate jobs?

Reply:
It's more for character building per my comment at the top: https://github.com/endless-sky/endless-sky/pull/3235#issuecomment-345553796

It also means you can condition a mission - even without #3128 - easily on total quantity without needing a massive chain where you sum all mission name conditions and subtract all the ones you failed, etc.

So it's not used - yet - but certainly could inspire content or other ideas.